### PR TITLE
Fix incorrect 'pass' parsing when more than one entry matches gitlab.com.

### PR DESCRIPTION
**Pull request by kacf**
Changelog: None

Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

**Comment by kacf** [comment]
Cancelling pipeline. This is not going to affect it.

**Comment by kacf** [comment]
@lluiscampos: It was inevitable it would happen at some point! :-)

File: extra/release_tool.py [hunk 1]
```diff
@@ -237,18 +234,34 @@ def get_value_from_password_storage(server, key):
...
-        if count == 0:
-            return
+        pass_dir = os.getenv("PASSWORD_STORE_DIR")
+        if not pass_dir:
+            pass_dir = os.path.join(os.getenv("HOME"), ".password-store")
+
+        server_path_str = os.getenv("PASS_GITLAB_COM")
+
+        if not server_path_str:
+            output = subprocess.check_output(["find", pass_dir, "-type", "f", "-path", "*%s*" % server]).decode()
```
Comment (lluiscampos) [inline extra/release_tool.py:244]: Aha! So using `find -path` is what I was missing when I did the first implementation. I couldn't figure out how to parse the tree like output of pass...

Thanks!

Comment (kacf) [reply extra/release_tool.py:244]: Hehe yes, this was after several iterations of frustratingly trying to parse `pass` output. It really has a very difficult to parse format, `find` is much easier.

**Pull request merged**